### ISSUE TITLE
Shipping costs on refunds

### ISSUE THREAD

**Issue by tommiby** [woocommerce/woocommerce-rest-api#134]
Hello,

it seems that refunfded shipping costs are not showing on order refunds when refund is accessed through REST API. The refund shows only the refunded order rows but not the shipping cost details when the shipping costs had been refunded. Would it be possible to add the the refunded shipping cost data also to refunds similar way as it is the orders?

**Comment by sponiza** [comment]
I agree with this one. Without the shipping tax details it is very hard to work with the refunds. Would appreciate if this could be picked up.
